templates/node-template/node-template.js

```diff
@@ -3,7 +3,7 @@ module.exports = function(RED) {
 
   function NodeTemplateNode(config) {
     RED.nodes.createNode(this, config);
-    const node = this;
+    // const node = this;
```

Comment:
```suggestion
    const node = this;
```
I think commenting this out was unintentional?